docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudtrail-firehose.asciidoc

```diff
@@ -0,0 +1,207 @@
+
+IMPORTANT: Make sure the deployment is on AWS, because the Amazon Data Firehose delivery stream connects specifically to an endpoint that needs to be on AWS.
+
+[discrete]
+[[firehose-cloudtrail-step-one]]
+== Step 1: Install AWS integration in {kib}
+
+. Install AWS integrations to load index templates, ingest pipelines, and dashboards into {kib}. In {kib}, navigate to *Management* > *Integrations* and find the AWS Integration by browsing the catalog.
+
+. Navigate to the *Settings* tab and click *Install AWS assets*. Confirm by clicking *Install AWS* in the popup.
+
+. Install AWS Firehose integration assets in Kibana. 
+
+NOTE: Amazon Data Firehose integration is currently in beta. Make sure to enable *Display beta integrations*.
+
+[discrete]
+[[firehose-cloudtrail-step-two]]
+== Step 2: Export Cloudtrail events to CloudWatch
+
+image::firehose-cloudtrail-cloudwatch.png[Cloudtrail to CloudWatch]
+
+To export CloudTrail logs to CloudWatch, you must set up a *trail* through the following steps:
+
+. Go to the https://console.aws.amazon.com/[AWS console] and navigate to CloudTrail.  
+
+. Click *Create trail* and configure the general details on the *Choose trail attributes* panel, like for example:
++
+* Trail name
+* Storage location
+
+By default, CloudTrail exports data to an S3 bucket. It isn’t possible to opt-out from S3.
+
+. Specify the encryption options.
++
+When exporting data from CloudTrail to S3, it is recommended to enable
+*Log file SSE-KMS encryption*. You can use an existing AWS KMS key, or create a new one.
+
+. Enable *CloudWatch Logs* and confirm the *Log group name*.
++
+CloudTrail offers the option to send events to CloudWatch as logs. You
+must enable this option to forward the events to Amazon Data Firehose.
++
+You also need to create an IAM Role, or select an existing one, to enable CloudTrail to put log events into a CloudWatch stream.
+
+. From the *Choose log events* panel, select the event types you want to send to Elastic.
+
+. Review the attributes and log events you have specified in the previous steps and click *Create trail*.
+
+. Verify everything is working as expected.
++
+Open the log group you created at Step 1 on CloudWatch and make sure there are events from the CloudTrail you have just created.
++
+image::firehose-verify-events-cloudwatch.png[Verify events in CloudWatch]
+
+[discrete]
+[[firehose-cloudtrail-step-three]]
+== Step 3: Set up a Firehose delivery stream
+
+image::firehose-delivery-stream.png[Firehose delivery stream]
+
+You now have a CloudWatch log group with events coming from CloudTrail.
+For more information on how to set up a Amazon Data Firehose delivery stream to send data to Elastic Cloud, you can also check the <<monitor-aws-firehose,setup guide>>.
+
+. Collect {es} endpoint and API key from your deployment on Elastic Cloud.
```

Comment:
Should we link to somewhere that explains where to find these? Or is it safe to assume a user will know where to find these?

Reply:
Added more details with f39a6e131bf7fba96c0b0d329d9d673308b28e6f.